okhttp/api/okhttp.api

```diff
@@ -68,6 +68,7 @@ public final class okhttp3/Cache : java/io/Closeable, java/io/Flushable {
 	public final fun -deprecated_directory ()Ljava/io/File;
 	public fun <init> (Ljava/io/File;J)V
 	public fun <init> (Lokio/Path;JLokio/FileSystem;)V
+	public fun <init> (Lokio/Path;JLokio/FileSystem;Lokhttp3/internal/concurrent/TaskRunner;)V
```

Comment:
I’m very anxious about `internal` stuff leaking into here

Reply:
Yep, this file is part of the review.  I'll check after these review updates.

Reply:
We could make it fail if there are entries here.